helm/reana/README.md

```diff
@@ -9,34 +9,38 @@ This Helm automatically prefixes all names using the release name to avoid colli
 |----------------------------------------------------------|--------------------------------------------------------------------------------------|-------------------------------------------------|
 | `components.reana_db.enabled`                            | Instantiate a PostgreSQL database inside the cluster                                 | true                                            |
-| `components.reana_job_controller.image`                  | [REANA-Job-Controller image](https://hub.docker.com/repository/docker/reanahub/reana-job-controller) to use  | `reanahub/reana-job-controller:<chart-release-version>` |
-| `components.reana_message_broker.image`                  | [REANA-Message-Broker image](https://hub.docker.com/repository/docker/reanahub/reana-message-broker) to use | `reanahub/reana-message-broker:<chart-release-version>` |
+| `components.reana_job_controller.image`                  | [REANA-Job-Controller image](https://hub.docker.com/r/reanahub/reana-job-controller) to use  | `reanahub/reana-job-controller:<chart-release-version>` |
+| `components.reana_message_broker.image`                  | [REANA-Message-Broker image](https://hub.docker.com/r/reanahub/reana-message-broker) to use | `reanahub/reana-message-broker:<chart-release-version>` |
 | `components.reana_message_broker.imagePullPolicy`        | REANA-Message-Broker image pull policy                                               | IfNotPresent                                    |
 | `components.reana_server.environment`                    | REANA-Server environment variables                                                   | {REANA_MAX_CONCURRENT_BATCH_WORKFLOWS: 30}      |
-| `components.reana_server.image`                          | REANA-Server image to use                                                            | `reanahub/reana-server:<chart-release-version>` |
+| `components.reana_server.image`                          | [REANA-Server image](https://hub.docker.com/r/reanahub/reana-server) to use          | `reanahub/reana-server:<chart-release-version>` |
 | `components.reana_server.imagePullPolicy`                | REANA-Server image pull policy                                                       | IfNotPresent                                    |
-| `components.reana_ui.enabled`                            | Instantiate the [REANA-UI](https://github.com/reanahub/reana-ui)                     | false                                           |
-| `components.reana_ui.image`                              | [REANA-UI image](https://hub.docker.com/repository/docker/reanahub/reana-ui) to use  | `reanahub/reana-ui:<chart-release-version>`     |
+| `components.reana_server.uwsgi.processes`                | Number of uWSGI proccesses                                                           | 6                                               |
```

Comment:
`s/proccesses/processes/g`

Reply:
Also, WRT commit headline `global: multiple amendments`, this is all about Helm, so it's better to say `helm: ....`.  You can even squash the two commits together.